boards/shields/mikroe_eth_click/boards/lpcxpresso55s69_cpu0.overlay

```diff
@@ -9,6 +9,11 @@
 &mikrobus_spi {
 	status = "okay";
 
+	cs-gpios = <&gpio0 20 GPIO_ACTIVE_LOW>,
+			   <&gpio1 1 GPIO_ACTIVE_LOW>,
+			   <&gpio1 12 GPIO_ACTIVE_LOW>,
+			   <&gpio1 26 GPIO_ACTIVE_LOW>;
```

Comment:
Looks off again.
Please use 8 chars tab and complete with for fine tuning.

Reply:
Ok, fixed and followed the coding style guide. Thanks for the heads up.